newIDE/app/src/UI/List.js

```diff
@@ -222,6 +223,7 @@ export class ListItem extends React.Component<ListItemProps, ListItemState> {
             <MUIListItemText
               primary={props.primaryText}
               secondary={props.secondaryText}
+              style={{ margin: '0 auto' }}
```

Comment:
I'm worried of these changes - what happen if you don't do this?

Reply:
To start with, I adjusted the height of the TextField component in TextField.js, It worked in the browser but the text wasn't center on renaming the objects in the electron app as top was -7px. I inspected the ListItem and found that the position of ListItem was fixed and the MUIListItemText had marginTop and marginBottom of 4px's by default. Resulting in the extra height of the ListItem and overlapping with the next ListItem on renaming. So I thought it would be better to give marginTop/Bottom to 0 and gave TextField.js  top equal to 0, as it automatically centers the TextField in MUIListItem.
I can try again without using these changes to find if there's another possible way.